extensions/resource-deployment/src/test/validations.test.ts

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please put this file in the same folder structure as the tested class (`src/test/ui/validation/validation.test.ts`)

Reply:
ok then all existing test files should also be moved to mimic the 'src' structure. Should that change be its own PR?

Reply:
Yes, that can be done as a follow up.

Reply:
Moved